Library/Homebrew/formula_creator.rb

```diff
@@ -9,11 +9,14 @@ module Homebrew
   #
   # @api private
   class FormulaCreator
-    attr_reader :args, :url, :sha256, :desc, :homepage
+    attr_reader :url, :sha256, :desc, :homepage
     attr_accessor :name, :version, :tap, :mode, :license
 
-    def initialize(args)
-      @args = args
+    def initialize(name, version, fetch, head)
```

Comment:
Adding a `sig` here for Sorbet would be nice, too. Shout if you need help or guidance doing so here.

Reply:
Added types..